.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
       - run:
           name: Tests
-          command: pytest --cov .
+          command: pytest --ignore=cdk --cov .
```

Comment:
Does coverage also need to ignore tests?

Reply:
will add `.coveragerc` that should do this.